mocha/test/organisation.test.js

```diff
@@ -0,0 +1,70 @@
+const httpStatus = require("http-status");
+const APP_URL = process.env.MOCHA_URL;
+const apiEndPoints = require("../utils/apiEndpoints");
+const apiHelper = require("../utils/apiHelper");
+const validator = require("../utils/validators");
+
+let orgApiEndPoint = apiEndPoints.organisationsEndpoint;
+let orgId = "5fc7aa97c0cdcc003855f308";
+let invalidId = "5fba8756dd5c260038d223a3";
+let ownerId = "5fc7aa97c0cdcc003855f304";
+
+describe("GET organisations endpoint tests", () => {
+  it("sending GET request with endpoint, status : 200 OK with org details", async () => {
+    let response = await apiHelper.sendGETRequest(APP_URL, orgApiEndPoint);
+    validator.validateStatusCode(response, 200);
+  });
+
+  it("sending GET request with endpoint with valid path parameter id, status : 200 OK with details", async () => {
```

Comment:
I have an impression that here we test another endpoint: 
​/api​/organisations​/{organisationId}